<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <addaction name="actionStep"/>
    <addaction name="actionSetCircle"/>
    <addaction name="actionCreateMask"/>
    <addaction name="actionStart"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpenFile"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSetCircle">
   <property name="text">
    <string>Размер и координаты области суммирования...</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Открыть 1-й кадр...</string>
   </property>
  </action>
  <action name="actionCreateMask">
   <property name="text">
    <string>Создать маску</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Поехали!</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="text">
    <string>Задать шаг...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
